### PR TITLE
fitBounds: Handle "degenerate" bounds

### LINKED ISSUE #67: Pixel project matrix not invertible
Some of our customers are receiving the following error, crashing our site: 
screen shot 2018-04-26 at 10 04 28 am

I need to understand what would cause this error so that I can make sure to catch it before crashing, thanks.

### PR DESCRIPTION
**Pull request by ibgreen**
Fixes #67

### CODE REVIEW

**Comment by ibgreen** [comment]
Published as 6.2.0

File: docs/whats-new.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
 # What's New
 
+## v6.2
+
+Release date: TBD April, 2019
```
Comment (akre54) [inline docs/whats-new.md]: @ibgreen Looks like these dates / versions are wrong? https://github.com/uber-common/viewport-mercator-project/releases

File: test/spec/fit-bounds.spec.js [hunk 2]
```diff
@@ -83,3 +83,32 @@ test('WebMercatorViewport.fitBounds', (t) => {
...
+  const OPTIONS = {
+    height: 100,
+    width: 100,
+    bearing: 0,
+    pitch: 0,
+    zoom: 2
+  };
+
+  const viewport = new WebMercatorViewport(OPTIONS);
+  t.doesNotThrow(
```
Comment (akre54) [inline test/spec/fit-bounds.spec.js]: Worth adding a test for the new `maxZoom`/`minZoom` options too?

File: docs/whats-new.md (hunk 1 above)
Comment (ibgreen) [reply docs/whats-new.md]: Noted. I took care of that in the v6.2.0 publish commit (pushed to master).

@jckr is working on a gatsby website update, when published that will pick up the correct date.

File: test/spec/fit-bounds.spec.js (hunk 2 above)
Comment (ibgreen) [reply test/spec/fit-bounds.spec.js]: > Worth adding a test for the new maxZoom/minZoom options too?

Agreed. Would you consider doing the honors? :)

Comment (akre54) [reply test/spec/fit-bounds.spec.js]: Done! #100

**Pull request merged**